src/schema/search.ts

```diff
@@ -1,11 +1,33 @@
 import { IResolvers } from '@graphql-tools/utils';
 import { traceResolvers } from './trace';
 import { Context } from '../Context';
-import { postFeedback, SearchResultFeedback } from '../integrations';
+import {
+  getSessions,
+  postFeedback,
+  SearchResultFeedback,
+  SearchSession,
+  SearchSessionParams,
+} from '../integrations';
 import { ValidationError } from 'apollo-server-errors';
 import { GQLEmptyResponse } from './common';
 
+type GQLSearchSession = Pick<SearchSession, 'id' | 'prompt' | 'createdAt'>;
+type GQLSearchSessionParams = Pick<SearchSessionParams, 'limit' | 'lastId'>;
+
 export const typeDefs = /* GraphQL */ `
+  type SearchSession {
+    id: String!
+    prompt: String!
+    createdAt: String!
```

Comment:
Since the value is coming from an external service, we are unaware of the format for the returned date. Hence, made it a String for safety.

Reply:
It's better to set everything as Date. It should be ok, if not we will parse it.
External service should be treated just like a database when we approach designing the API